plugins/SLURM-Integration/install.sh

```diff
@@ -64,6 +64,19 @@ function update_slurm_conf()
     fi
 }
 
+function update_python_path_for_ufm_prolog_epilog()
+{
+    python_path=$(which python3)
+    sudo sed -i -e "s#python3_path#$python_path#g" $SLURM_DIR/$UFM_PROLOG_FILE
+    if [[ $? -ne 0 ]];then
+        echo "Warning:: Failed to replace the output of 'which pythn3' command with python3_path keyword in $SLURM_DIR/$UFM_PROLOG_FILE file, please try to do it manually."
```

Comment:
Failed to set PYTHONPATH for $filename (prolog / epilog), please set it manually at $filepath by setting "python3_path" parameter, prior to running the UFM-Slurm integration.